assignment/src/main/java/nerdschool/bar/Pub.java

```diff
@@ -1,69 +1,7 @@
 package nerdschool.bar;
 
+// A copy for commenting code
+	
 public class Pub {
 
-
-    public static final String ONE_BEER = "hansa";
-    public static final String ONE_CIDER = "grans";
-    public static final String A_PROPER_CIDER = "strongbow";
-    public static final String GT = "gt";
-    public static final String BACARDI_SPECIAL = "bacardi_special";
-
-    public int computeCost(String drink, boolean student, int amount) {
-
-        if (amount > 2 && (drink == GT || drink == BACARDI_SPECIAL)) {
-            throw new RuntimeException("Too many drinks, max 2.");
```

Comment:
A custom more explicit exception?
This exception should extend Exception (checked, need try/catch, frequent) or RuntimeException (unchecked) and why?
Example : throw new TooManyDrinksException(message);